### DISCUSSION TITLE
shift of analog ROI

### DISCUSSION THREAD

**Discussion by josefkozisek** [jomjol/AI-on-the-edge-device#3182] [Q&A]
Hi all,
my first Analog ROI is going about 0,3 ahead. For example first ROI is 1,9 , second ROI 6, together it is 0,16. But when second ROI increase about 3, go to 9 , first ROI exceed 2. For example first ROI 2,1, second  ROI 9. together is it 0,29. It is wrong , correct is 0,19.
Is any way how to shift analog ROI or how to manage it?
Thank you

**Comment by josefkozisek** [comment]
![Screenshot_20240816_014602_Samsung Internet](https://github.com/user-attachments/assets/e1169032-379c-4f79-a800-f1c1a74b96ba)

Just I add watermeter photo. You can see first ROI is 3,1 , but it must be still 2. correct is 230,27, but system read it as 230,37. It create error. I would like shift first ROi back a little.
Thank you

**Comment by josefkozisek** [comment]
Mark, I think recognising of values looks perfect, you can see Recognized analogue ROI. It is exactly what is there. Unfortunately my first ROI go ahead about 0,2-0,3, I would like to correct this deviations somewhere? Thank you

**Reply by caco3** [reply]
There is currently no way to tune a single ROI.
The way the analog ROI models work is by calculating the angle out of the image. So maybe you can improve the situation by moving the ROI a bit. The best is if you center it around the center of the pointer (so a few pixel to the right and top). Don't care about the surounding circle and values of the display.
